core/dbt/events/functions.py

```diff
@@ -110,15 +110,14 @@ def cleanup_event_logger():
     EVENT_MANAGER.callbacks.clear()
 
 
-# The default event manager will not log anything, but some tests run code that
-# generates events, without configuring the event manager, so we create an empty
-# manager here until there is a better testing strategy in place.
+# Since dbt-rpc does not do its own log setup, and since some events can
+# currently fire before logs can be configured by setup_event_logger(), we
+# create a default configuration with default settings and no file output.
 EVENT_MANAGER: EventManager = EventManager()
+EVENT_MANAGER.add_logger(
```

Comment:
would [adding additional loggers downstream](https://github.com/dbt-labs/dbt-core/blob/60f80056b1b09b35a22ac91c93319827bdac0e0e/core/dbt/main.py#L234) result in [extraneous calls to `write_line`](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/events/eventmgr.py#L194-L196), potentially to the same logger instance if it is added more than once to the event manager?